doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -31,6 +31,9 @@ The following sections provide detailed lists of changes by component.
 Application development
 =======================
 
+* Implemented new generic configuration scheme for build types in application development and updated several applications accordingly.
```

Comment:
We can't say implemented.

We could say "Implemented partition manager support for new build type configuration scheme and updated several applications accordinlgy."

Reply:
"The new scheme is based on Zephyr and uses application-specific :file:prj.conf files instead of build types selected through CMake build type variable (CMAKE_BUILD_TYPE)."

change to

"The new scheme infers the build type from CONF_FILE instead of using the CMake variable CMAKE_BUILD_TYPE."